python/cugraph/cugraph/gnn/graph_store.py

```diff
@@ -15,6 +15,9 @@
 import cugraph
 from cugraph.experimental import EXPERIMENTAL__PropertyGraph as PropertyGraph
```

Comment:
you can do  from _cugraph.experimental import PropertyGraph_

Reply:
I just changed it.